stories/css-shadow-variable-clash.tsx

```diff
@@ -0,0 +1,14 @@
+import { styled } from '@compiled/react';
+
+export default {
+  title: 'css prop/variable clash',
+};
+
+const isPrimary = true;
+const Component1 = styled.p`
+  color: ${({ isPrimary }) => (isPrimary ? 'green' : 'red')};
+`;
+
+export const UsingComponent1 = (): JSX.Element => {
+  return <Component1 isPrimary={false}>Component1</Component1>;
```

Comment:
Can you
- use a more descriptive component name, eg ComponentWithProps
- add a more descriptive text, eg 'This should be red'

Reply:
Updated the component name and it's content